src/main/java/com/microsoft/aad/msal4j/UserNamePasswordParameters.java

```diff
@@ -46,6 +46,10 @@
     @NonNull
     private char[] password;
 
+    public char[] password(){
```

Comment:
>password [](start = 18, length = 8)

ClonePassword?

Reply:
to keep it immutable we should not expose internal representation which is array, so mutable